src/gameplaykit.cs

```diff
@@ -48,7 +77,7 @@ interface GKAgent {
 		float Radius { get; set; }
 
 		[Export ("speed")]
-		float Speed { get; }
+		float Speed { get; [iOS (10,0), TV(10,0), Mac (10,12, onlyOn64: true)] set; }
```

Comment:
@spouliot could this be a breaking change?

The generated `Speed` property will be virtual, which means it can be overridden.

If a binary is overriding a virtual readonly property, and then the base property becomes read-write, the override will be wrong since it won't have the setter. I'm not sure mono will complain about this though.